src/chainparams.cpp

```diff
@@ -121,7 +121,6 @@ class CMainParams : public CChainParams {
 
         genesis = CreateGenesisBlock(1231006505, 2083236893, 0x1d00ffff, 1, 50 * COIN);
         consensus.hashGenesisBlock = genesis.GetHash();
-        assert(consensus.hashGenesisBlock == uint256S("0x000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f"));
```

Comment:
If it was really optional you shouldn't need to remove this.

Reply:
It hits this assert if I don't move it, for some reason. I was hacking to get it working, will re-investigate now that I have it working